prep-javascript-functions/main.js

```diff
@@ -0,0 +1,61 @@
+function addTwoNumbers(num1, num2) {
+  return num1 + num2;
+}
+
+var addTwoNumbersResult = addTwoNumbers(2, 5);
+console.log(addTwoNumbersResult);
+
+function convertHoursToMinutes(hours) {
+  return 4 * 60;
+}
+var convertHoursToMinutesResult = convertHoursToMinutes(4);
+console.log(convertHoursToMinutesResult);
+
+function getGreeting(name) {
+  return 'Hello World';
+}
+
+var getGreetingResult = getGreeting(name);
+console.log(getGreetingResult);
+
+function addAndMultiplyBy5(num1, num2) {
+  return num1 + num2;
+}
+
+var addAndMultiplyby5Result = addAndMultiplyBy5(10, 5);
+console.log(addAndMultiplyby5Result * 5);
+
+function multiplyAndDivideBy5(num1, num2) {
+  return num1 * num2;
+}
+
+var multiplyAndDivideBy5Result = multiplyAndDivideBy5(35, 10);
+console.log(multiplyAndDivideBy5Result / 5);
+
+function subtractTwoNumbers(num1, num2) {
+  return num1 - num2;
+}
+
+var subtractTwoNumbersResult = subtractTwoNumbers(35, 10);
+console.log(subtractTwoNumbersResult);
+
+function getCircleCircumference(radius) {
+  return 2 * Math.PI * 5;
+}
+
+var getCircleCircumferenceResult = getCircleCircumference(5);
+console.log(getCircleCircumferenceResult);
+
+function cube(number) {
+  return 5 * 5 * 5;
+}
+
+function getFullName(firstName, lastName) {
+  return 'Anthony' + ' ' + 'Pineda';
+}
```

Comment:
Similarly with these functions, try using your parameters so that the functions can work with different values.